.template/addons/bootstrap/stylesheets/bootstrap/bootstrap.scss.tt

```diff
@@ -0,0 +1,42 @@
+// By default every component is imported
+// But DO NOT import the whole framework but instead
+// pick what the project requires
+// and comment out the rest.
+
```

Comment:
Can we just `@import 'bootstrap/scss/bootstrap'`? since in bootstrap.js file import all of this already.

Or not sure if we can `@import 'bootstrap/scss/bootstrap'` in `application.scss` directly?

Reply:
Of course we could do that. But if you look at the comment in top of the file and the rule we have regarding using frameworks in [Compass](https://compass.nimblehq.co/development/code-conventions/css/#frameworks) -

> But frameworks also have downsides such as file size and unused code among others. So to protect the application from these downsides, our usage of frameworks comes with restrictions and conventions.
> 
> Do NOT import the whole framework but instead pick what the project requires.

So, I imported them like this so that we can use what we need and comment out the rest.

Reply:
Make sense. You can go with your solution.